dot/node.go

```diff
@@ -177,13 +176,6 @@ func LoadGlobalNodeName(basepath string) (nodename string, err error) {
 
 // NewNode creates a new dot node from a dot node configuration
 func NewNode(cfg *Config, ks *keystore.GlobalKeystore) (*Node, error) {
-	// set garbage collection percent to 10%
-	// can be overwritten by setting the GOGC env variable, which defaults to 100
-	prev := debug.SetGCPercent(10)
-	if prev != 100 {
-		debug.SetGCPercent(prev)
-	}
-
```

Comment:
why is this removed? does it not affect memory usage anymore?

Reply:
I would like to remove it to have a clear picture. I don't really think it helps much anyway, the GC is pretty optimized. And also our memory problem is not GC related. Is there a reason we added it in the first place?

Reply:
the memory usage was way worse without it, but we can remove it and just set it if needed

Reply:
We can always set it with `GOGC` if you need to run the node for as long as you want.
Although I guess setting to 10 will bottleneck the CPU so the program will run slower, hence the longer time perhaps? 🤔 
From my point of view, I prefer to have it run as fast as possible and crash to see if my changes fixed it or not 😄 So I guess I can also use `GOGC`, up to you I can also revert it!

Reply:
no, it's fine to delete it, we can just re-add it later if we want